src/endpoint/s3/ops/s3_get_bucket.js

```diff
@@ -95,6 +112,40 @@ function key_marker_to_cont_tok(key_marker, objects_arr, is_truncated) {
     return Buffer.from(j).toString('base64');
 }
 
+function set_encoding_flag(encoding_type) {
```

Comment:
`set_***()` usually means we are setting a value and storing it somewhere,
so `get_encoding_type` is actually more appropriate here

Reply:
I started with `get_encoding_flag` since it is a return from this function, and then I changed it to `set_encoding_flag `because it changes the flag. Since it is not an OOP and the 'setter' and the 'getter' are not the classic ones, I will change this name to `init_encoding_flag`.

Reply:
"init" also implies you are modifying something like "set", so still "get" makes more sense. but if you really insist then pethaps parse_.. or check_...